keyboards/redragon/k630/info.json

```diff
@@ -0,0 +1,190 @@
+{
+    "keyboard_name": "K630",
+    "manufacturer": "Redragon",
+    "url": "https://www.redragonzone.com/products/redragon-k630-gaming-mechanical-keyboard",
+    "maintainer": "bausensi",
+    "bootloader": "sn32-dfu",
+    
+    "processor": "SN32F248BF",
+    "usb": {
+        "vid": "0x0C45",
+        "pid": "0x5004",
+        "device_version": "0.0.1",
+        "force_nkro": true
+    },
+
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+
+    "debounce": 3,
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": [ "A0",  "A1",  "A2",  "A3", "A4", "A5",  "A6",  "A7",  "A8", "A9", "A10", "A11", "A12", "A13" ],
+        "rows": [ "D11", "D10", "D9",  "D8", "D7" ]
+    },
+    
+    "tapping": {
+        "term": 150
+    },
+
+    "indicators": {
+        "caps_lock": "B14",
+        "on_state": 0
+    },
+    
+    "rgb_matrix": {
+        "driver": "SN32F24xB",
+        "hue_steps": 8,
+        "sat_steps": 8,
+        "val_steps": 8,
+        "speed_steps": 8,
```

Comment:
these seem to be here for convenience and not a hardware reason. The shared matrix driver supports 8/16/16/16 as default which is higher resolution. Probably best to limit those to userspace/keymap territory if you want convienience. As a rule of thumb, we want the keyboards in here to be as close to stock behavior as possible in the main configuration.